docs/src/pages/configurations/typescript-config/index.md

```diff
@@ -47,7 +47,7 @@ The above example shows a working Webpack config with the TSDocgen plugin also i
 {
   "compilerOptions": {
     "outDir": "build/lib",
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
i believe if we're targeting `es5` then module generation is using `commonjs` according to typescript docs

https://www.typescriptlang.org/docs/handbook/compiler-options.html

Can you post a reproduction?

Reply:
@wuweiweiwu just tried those values:

```
    "module": "commonjs",
    "target": "es5",
```

and getting an error tough it works with the previous values (esnext and esnext for both module and target)

Reply:
I'm not sure what's right, we're using all of the above in the codebase...

![screenshot 2019-02-14 at 17 07 47](https://user-images.githubusercontent.com/3070389/52799919-11d82780-307b-11e9-8c48-ad59fae22bc9.png)

Lets get some more typescript folk in here..

@kroeder @gaetanmaisse @apastuhov 

What do you guys thinks is the correct value for our docs?

Reply:
@canercandan Do you use dynamic module imports in your project? 

I did not succeed in reproducing your problem neither using React sample project nor a real world angular app. Can you create a minimal example reproducing this issue?

---

@ndelangen I will investigate in the next few days about the different values we are using in SB codebase.

Reply:
Thanks, I'll close this PR in the meantime then. Please re-open if this is something we want to continue